src/openrct2/rct1.h

```diff
@@ -721,8 +723,8 @@ typedef struct rct1_s4 {
     rct_ride_measurement ride_measurements[8];
     uint32 next_guest_index;
     uint16 game_counter_5;
-    uint8 patrol_areas[0x3C00];
-    uint8 unk_1F42AA[116];
+    uint8 patrol_areas[(RCT1_MAX_STAFF + STAFF_TYPE_COUNT) * RCT12_PATROL_AREA_SIZE];
```

Comment:
Needs a separate RCT12_STAFF_TYPE_COUNT, in case OpenRCT2 ever supports more types of staff (either directly or through plug-ins).